recipe/meta.yaml

```diff
@@ -7,17 +7,18 @@ package:
 source:
   fn: cmt-v{{ version }}.tar.gz
   url: https://github.com/lucastheis/cmt/archive/v{{ version }}.tar.gz
-  sha1: 7482caefa5359a351ac834920a092b5594098d70
+  sha256: 252d1029c62889a533d50e017445407e1de653cac8bee48ecdc34e4d8fd6e7ce
   patches:
-    # Workaround to ensure `gcc` compiler is used on Mac.
-    # See issue ( https://github.com/lucastheis/cmt/issues/10 ).
-    # No patch submitted as this will require discussion with the developer.
+    # Patch to avoid passing `-stdlib` to the compiler generally
+    # See issue ( https://github.com/lucastheis/cmt/issues/25 ).
+    # Patch for this has been submitted.
+    # See pull request ( https://github.com/lucastheis/cmt/pull/27 ).
     - setup.py.patch
-    # Incorrect import here makes tests fail and reduces functionality.
-    # See issue ( https://github.com/lucastheis/cmt/issues/14 ).
+    # Patch to register missing Python method added in 0.6.0.
+    # See issue ( https://github.com/lucastheis/cmt/issues/29 ).
     # Patch for this has been submitted.
-    # See pull request ( https://github.com/lucastheis/cmt/pull/18 ).
-    - images.py.patch
+    # See pull request ( https://github.com/lucastheis/cmt/pull/30 ).
+    - module.cpp.patch
```

Comment:
Opened issue ( https://github.com/lucastheis/cmt/issues/29 ) about the problem and PR ( https://github.com/lucastheis/cmt/pull/30 ) with a proposed fix. This affects part of the Python API that was determined to be missing in the tests. As the rest of the code is already there, was a simple matter to register this method for Python, which fixed the issue.